lib/sinatra/authorization.rb

```diff
@@ -40,6 +55,27 @@ def authorize_with_ssl_client
         end
       end
 
+      # Common set of authorization methods used in foreman-proxy
+      def do_authorize_any
```

Comment:
IMHO any is a bit ambiguous. Any could also mean authorized any client without authentication. Perhaps `do_authorize_any_method` would be clearer but naming is hard.

Reply:
Yeah I was thinking maybe `do_authorize_all` because we actually do not authorize but rather refuse unauthorized requests (the opposite). It's complicated, I was thinking like `any` reads a bit cleaner in a sense "any method will do it".